debian/control

```diff
@@ -0,0 +1,28 @@
+Source: libplayback
+Section: libs
+Priority: optional
+Maintainer: Ivaylo Dimitrov <ivo.g.dimitrov.75@gmail.com>
+Build-Depends: debhelper (>= 5), libtool, pkg-config, libdbus-1-dev
+
+Package: libplayback-1-0
+Architecture: any
+Section: libs
+Priority: optional
+Depends: ${shlibs:Depends}
+Description: OSSO Playback client library.
+
+Package: libplayback-1-dev
+Architecture: any
+Section: devel
+Priority: optional
```

Comment:
If Section or Priority is not specified in Package part, then it is automatically taken from the Source part. So it does not make sense to re-define Section or Priority in Package part to same value.

I would say, that we should match what Maemo binaries have (if values are not bogus).